templates/web/common/site_layout.tt.html

```diff
@@ -411,6 +411,8 @@ <h3 class="title-5 text-medium">[% lang('footer_discover_the_project') %]</h3>
 <script src="[% static_subdomain %]/js/dist/display.js"></script>
 <script src="[% static_subdomain %]/js/dist/stikelem.js"></script>
 <script src="[% static_subdomain %]/js/dist/scrollNav.js"></script>
+<script src="[% static_subdomain %]/js/dist/togglepassword.js"></script>
```

Comment:
Those 2 scripts are only needed on the sign-in and user page, can you load them only on those pages instead of all pages of the website?